Offseason2022/src/main/java/frc/robot/commands/Climber8SettleToL4.java

```diff
@@ -13,17 +17,22 @@ public class Climber8SettleToL4 extends SequentialCommandGroup
 {
   public Climber8SettleToL4(Climber climber)
   {
+    setName("ClimberSettleToL4");
+
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+
+        // @formatter:off
+        new ParallelDeadlineGroup(
+            new WaitUntilCommand(climber::moveClimberDistanceIsFinished),
+            new ClimberMoveToHeight(climber, Height.STOW_HEIGHT), 
+        new ClimberSetGatehook(climber, false)),

```

Comment:
ClimberSetGateHook gets indented, and notice the extra parens on this line. That means the three previous commands are in the first ParallelDeadlineGroup and should have the same indent level. The rest of the commands in here are fine.